src/status_im/utils/logging/core.cljs

```diff
@@ -4,15 +4,36 @@
             [status-im.utils.fx :as fx]
             [status-im.utils.types :as types]
             [status-im.utils.handlers :as handlers]
-            [status-im.utils.email :as mail]))
+            [status-im.utils.email :as mail]
+            [taoensso.timbre :as log]
+            [status-im.utils.config :as config]))
 
 (def report-email "error-reports@status.im")
+(def max-log-entries 1000)
+(def logs-queue (atom #queue[]))
+(defn add-log-entry [entry]
+  (swap! logs-queue conj entry)
+  (when (>= (count @logs-queue) max-log-entries)
+    (swap! logs-queue pop)))
```

Comment:
It always conj and then pop by condition.
Why not just conj only when count < max?

Reply:
Oh, nevermind. I was wrong, it works like queue here.